### ISSUE TITLE
Pop out a view?

### ISSUE THREAD

**Issue by TheMikeyRoss** [responsively-org/responsively-app#1262]
Is there a way to pop up a device to an external window?

**Comment by MdFaizu18** [comment]
Yeah To "pop up a device to an external window" , we can achieve this using JavaScript within your React frontend. By Below method

`PopButton.jsx`
import React from 'react';

const PopupButton = () => {
  const openPopup = () => {
    const url = 'https://example.com'; 
    const windowName = 'newWindow'; 
    const windowFeatures = 'width=800,height=600'; 

    window.open(url, windowName, windowFeatures);
  };

  return (
    <button onClick={openPopup}>
      Open Popup
    </button>
  );
};

export default PopupButton;

**Comment by TheMikeyRoss** [comment]
That feels like an AI response. 

I mean that in the ResponsivlyApp I have multiple devices views (iPhone, Android, Desktop) and they are all next to each other and I have to scroll around to see each device.

But I want to pop out one of those devices and move that window to a different monitor so I can see them each in a monitor.

**Comment by violetadev** [comment]
hi @TheMikeyRoss, we don't have this feature (yet!). I have labeled this issue so anyone can pick it up and add it whenever possible.

**Comment by violetadev** [comment]
@CarlosZBent we're not assigning issues anymore, you can just go ahead and prepare the PR. Thanks!
